examples/pitchmelodia-rt/index.html

```diff
@@ -28,7 +28,7 @@ <h2 class="ui header white-text" style="color: azure;">
                         Real-time pitch extraction with Essentia.js
                     </h2>
                     <div class="ui basic large button">
-                        <a href="https://github.com/MTG/essentia.js/tree/master/examples/realtime/melspectrogram/example.html" target="_blank" class="ui small button">Code<i class="right github icon"></i></a>
+                        <a href="https://github.com/MTG/essentia.js/tree/dev/examples/pitchmelodia-rt/" target="_blank" class="ui small button">Code<i class="right github icon"></i></a>
```

Comment:
Add relative paths to the HTML here instead of the full URL.  Full paths require us to always change the base URI according to the branch we are in. Having relative paths makes it agnostic to these changes. Eg. ```./pitchmelodia-rt/```

Reply:
I might be wrong, but I don't think we can solve having these URIs be branch-agnostic by using relative paths. These are meant to be links to the code, whereas relative paths will refer to the GitHub pages domain it's being served from. Doing `./pitchmelodia-rt/` will break with a 404, since the resulting URI would be https://mtg.github.io/essentia.js/examples/pitchmelodia-rt/pitchmelodia-rt. And doing something like `./` will just point you to the page you're in. 

We would have to know on the HTML end what branch the GitHub pages is being built from, so that we could have links of the type `https://github.com/MTG/essentia.js/tree/<branch-of-current-pages-built>/examples/<demo-name>/`